api_server.py

```diff
@@ -202,9 +210,22 @@ def on_get(self, req, resp, client, start_slot, end_slot=None):
 
 classifier = None
 if not DISABLE_CLASSIFIER:
-    print("Initialising classifier, this could take a moment...")
-    classifier = MultiClassifier(DATA_DIR) if not DISABLE_CLASSIFIER else None
-    print("Done")
+    if MODEL_PATH != "":
+        if MODEL_PATH.endswith(".pkl"):
+            classifier = import_classifier(MODEL_PATH)
+
+        else:
+            print("model path must end with .pkl")
+            exit(0)
+
+    else:
+        print("Initialising classifier, this could take a moment...")
+        classifier = MultiClassifier(DATA_DIR) if not DISABLE_CLASSIFIER else None
+        print("Done")
+
+if classifier is None:
+    print("The classifier was not loaded")
+    exit(0)
```

Comment:
I'm going to delete this because it conflicts with the intention of `DISABLE_CLASSIFIER`. Sometimes it is useful to start the API server without a classifier, e.g. if you just want to serve info from the sqlite DB.

If we want to simplify this in future and always require a classifier, we should remove the `DISABLE_CLASSIFIER` option

```suggestion
```